crafters/nlp/TikaExtractor/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM logicalspark/docker-tikaserver
+
+WORKDIR /
+
+# get python
+RUN apt-get update && apt-get upgrade -y && apt-get clean
```

Comment:
Can u put all these layers together so that not so many partial images are created? This helps to obtain a smaller docker image

Reply:
Done